grammar.js

```diff
@@ -377,9 +377,9 @@ module.exports = grammar({
     'extras': $ => [
         /[ \t]+/,
         $.nl,
-        $.minor_comment,
         $.zeekygen_head_comment,
         $.zeekygen_prev_comment,
         $.zeekygen_next_comment,
+        $.minor_comment,
```

Comment:
I think the order in which these matchers are listed has consequences (also not the order in which rules are declared above), even though this change makes it look like it does.